src/order_status.cpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "..\include\TradingEngine\order_status.h"
+
+#include<unordered_map>
+using std::unordered_map;
+
+#include <string>
+using std::string;
+
+unordered_map<OrderStatus, string> order_status_map = {
+	{OrderStatus::NEW_ORDER, "New Order"},
```

Comment:
Make the values the same as the keys as well, so:
```
{OrderStatus::NEW_ORDER, "NEW_ORDER"},
```

Having spaces is a bad practice in dictionaries/maps.

Reply:
fixed, done